src/assets/img/dot-logo.svg

```diff
@@ -1,4 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 170 170" style="enable-background:new 0 0 170 170" xml:space="preserve">
+<svg xmlns="http://www.w3.org/2000/svg" style="enable-background:new 0 0 170 170" xml:space="preserve" viewBox="47.2 34.7 75.6 99.53">
```

Comment:
This update is to trim the whitespace from the logo to make it easier to style and position consistently.